File: concrete/src/main/java/pcb/uwu/unit/base/KelvinUnit.java
package pcb.uwu.unit.base;

import pcb.uwu.units.base.TemperatureUnit;

import java.util.function.Function;

public class KelvinUnit extends TemperatureUnit {

	public static final KelvinUnit KELVIN = new KelvinUnit();

	private KelvinUnit() {
		super("K", "kelvin", "kelvins",
				Function.identity(),
				Function.identity());
	}
}

File: concrete/src/main/java/pcb/uwu/unit/base/KelvinUnit.kt
package pcb.uwu.unit.base

import pcb.uwu.util.identity

object KelvinUnit
    : TemperatureUnit(symbol = "K",
                      singularName = "kelvin",
                      pluralName = "kelvins",
                      translationToCanonical = identity(),
                      translationFromCanonical = identity())
